contracts/system_tests/helpers/TestBaseWorkflow.sol

```diff
@@ -16,6 +16,7 @@ import '../../JBProjects.sol';
 import '../../JBSplitsStore.sol';
 import '../../JBToken.sol';
 import '../../JBTokenStore.sol';
+import '../../JBNFTRewardDataSourceDelegate.sol';
```

Comment:
don't you mind moving this import into the test itself? Not part of the base per se